apps/meteor/app/livechat/server/methods/registerGuest.js

```diff
@@ -32,14 +33,15 @@ Meteor.methods({
 		});
 
 		if (customFields && customFields instanceof Array) {
+			// TODO: refactor to use normal await
 			customFields.forEach((customField) => {
 				if (typeof customField !== 'object') {
 					return;
 				}
 
 				if (!customField.scope || customField.scope !== 'room') {
 					const { key, value, overwrite } = customField;
-					LivechatVisitors.updateLivechatDataByToken(token, key, value, overwrite);
+					Promise.await(LivechatVisitors.updateLivechatDataByToken(token, key, value, overwrite));
```

Comment:
```suggestion
					await LivechatVisitors.updateLivechatDataByToken(token, key, value, overwrite);
```

Reply:
I kept this one as Promise.await to not refactor the foreach in this PR 😬